OnStreetParking/examples/example-normalized-ld.jsonld.csv

```diff
@@ -1,2 +1,2 @@
-"urn:ngsi-ld:OnStreetParking:santander:daoiz_velarde_1_5", "OnStreetParking", "2016-06-02T09:25:55.00Z", "Property", "blueZone", "shortTerm", "forDisabled", "Property", "Mo, Tu, We, Th, Fr, Sa 09:00-20:00", "Property", "blueZonePermit", "disabledPermit", "Property", "car", "Property", "temporaryFee", "Relationship", "urn:ngsi-ld:ParkingGroup:daoiz-velarde-1-5-main", "urn:ngsi-ld:ParkingGroup:daoiz-velarde-1-5-disabled", "Property", "6", "GeoProperty", "Polygon", "-3.80356167695194", "43.46296641666926", "-3.803161973253841", "43.46301091092682", "-3.803147082548618", "43.462879859445884", "-3.803536474744068", "43.462838666196674", "-3.80356167695194", "43.46296641666926", "Property", "Zona Centro", "Property", "PT2H", "Property", "2", "Property", "3", "2018-09-12T12:00:00Z", "https://schema.lab.fiware.org/ld/context", "https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld"
\ No newline at end of file
+"id_", "type_", "modifiedAt_", "category__type_", "category__value__0_", "category__value__1_", "category__value__2_", "permitActiveHours__type_", "permitActiveHours__value__blueZonePermit_", "requiredPermit__type_", "requiredPermit__value__0_", "requiredPermit__value__1_", "allowedVehicleType__type_", "allowedVehicleType__value_", "chargeType__type_", "chargeType__value__0_", "refParkingGroup__type_", "refParkingGroup__object__0_", "refParkingGroup__object__1_", "totalSpotNumber__type_", "totalSpotNumber__value_", "location__type_", "location__value__type_", "location__value__coordinates__0__0__0_", "location__value__coordinates__0__0__1_", "location__value__coordinates__0__1__0_", "location__value__coordinates__0__1__1_", "location__value__coordinates__0__2__0_", "location__value__coordinates__0__2__1_", "location__value__coordinates__0__3__0_", "location__value__coordinates__0__3__1_", "location__value__coordinates__0__4__0_", "location__value__coordinates__0__4__1_", "areaServed__type_", "areaServed__value_", "maximumAllowedStay__type_", "maximumAllowedStay__value_", "extraSpotNumber__type_", "extraSpotNumber__value_", "availableSpotNumber__type_", "availableSpotNumber__value_", "availableSpotNumber__observedAt_", "occupiedSpotNumber__type_", "occupiedSpotNumber__value_", "occupiedSpotNumber__observedAt_", "@context__0_", "@context__1_"
```

Comment:
this file is generated automatically so it is not necessary (either convenient to be in the PR)